codespell_lib/data/dictionary.txt

```diff
@@ -3599,6 +3617,7 @@ foppy->floppy
 forbad->forbade
 forbiden->forbidden
 foreing->foreign
+forewarded->forwarded
```

Comment:
should come after the next entry. 
Also since foreward is british spelling you can add it as such:
forewarded->forewarded, forwarded,
I'll link this to https://github.com/lucasdemarchi/codespell/issues/103#issuecomment-341706499 because eventually I'd love to have a british and american feature flag.